janusgraph-all/pom.xml

```diff
@@ -49,11 +49,6 @@
             <version>${hbase098.core.version}-hadoop2</version>
         </dependency>
         <dependency>
-            <groupId>${project.groupId}</groupId>
-            <artifactId>janusgraph-hadoop</artifactId>
```

Comment:
I am a little confused here.
If we remove janusgraph-hadoop from the janusgraph-all pom, does it mean that an application that depends on janusgraph-all in its pom will not have janusgraph-hadoop artifact pulled in, thus missing this dependency?